src/main/kotlin/dev/arbjerg/ukulele/command/QueueCommand.kt

```diff
@@ -12,7 +12,7 @@ import org.springframework.stereotype.Component
 @Component
 class QueueCommand(
         private val players: PlayerRegistry
-) : Command("queue", "q", "list", "l") {
+) : Command("queue", "q", "list", "li") {
```

Comment:
For the new Loop (queue) command, I took the "l" since queue already had "q" as a one character shortcut.